src/main/java/org/sagebionetworks/bridge/sdk/Config.java

```diff
@@ -500,6 +544,20 @@ private String val(Props prop) {
         checkNotNull(value, "The property '" + prop.getPropertyName() + "' has not been set.");
         return value.trim();
     }
+    
+    private String val(Props prop, NameValuePair queryParam) {
+        return val(prop, Lists.newArrayList(queryParam));
+    }
+    
+    private String val(Props prop, List<NameValuePair> queryParams) {
+        try {
+            URIBuilder builder = new URIBuilder(val(prop));
+            builder.addParameters(queryParams);
+            return builder.build().toString();
```

Comment:
We're sure from the docs ("The parameter name and value are expected to be unescaped and may contain non ASCII characters.") I can write a test but this wouldn't work in the integration tests if it didn't escape email addresses.